.github/workflows/build-release-packages.yml

```diff
@@ -0,0 +1,104 @@
+name: Build Release Packages
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Are all tags always releases? Can we include only the ones starting with a digit?

Reply:
Yes we can, but don't we always release the tags only for releases?

Reply:
There are some tags which represent old redis releases. They're prefixed with "redis-" now. I kept them to be able to compare git history, etc. and they're not our releases.

Reply:
I can delete them otherwise. It seems brew also took them for releases...

Reply:
The older tags would not be affected by this, this is only applicable for pushing new tag

Reply:
I would do it based off a release, not a tag push, https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release instead. Like, I pushed a wrong tag, it seems more accurate to just do a release published. That would also include the tag, and would allow us to trigger everything together.